server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/model/DatabaseChangeParameters.java

```diff
@@ -42,7 +43,8 @@ public class DatabaseChangeParameters implements Serializable, TaskParameters {
     private List<String> rollbackSqlObjectIds;
     private Long timeoutMillis = 172800000L;// 2d for default
     private TaskErrorStrategy errorStrategy;
-    private String delimiter = ";";
+    @NotBlank
+    private String delimiter;
```

Comment:
delimiter parameter should has default value, 
if not set, then use `;` instead